bindings/python/test.py

```diff
@@ -266,6 +268,34 @@ def test_apply_settings(self):
 		self.assertEqual(s.get_settings()['num_want'], 66)
 		self.assertEqual(s.get_settings()['user_agent'], 'test123')
 
+class test_example_client(unittest.TestCase):
+
+	def test_execute_client(self):
+		with open(os.devnull, "w") as DEVNULL:
+			process = subprocess.Popen([sys.executable,"client.py","url_seed_multi.torrent"], stdout=DEVNULL, stderr=subprocess.STDOUT)
+		# python2 has no Popen.wait() timeout
+		time.sleep(10)
+		returncode = process.poll()
```

Comment:
I just want to know that it is still running after 10s to check that there were no bad interface calls. I guess 5s would be enoght